frb/data/FRBs/FRB190102.json

```diff
@@ -0,0 +1,28 @@
+{
+    "DM": {
+        "unit": "pc / cm3",
+        "value": 362.0
+    },
+    "DMISM": {
+        "unit": "pc / cm3",
+        "value": 56.815596175334385
+    },
+    "DM_err": {
+        "unit": "pc / cm3",
+        "value": 1.0
+    },
+    "FRB": "FRB190102",
+    "cosmo": "Planck15",
+    "dec": -79.4756388888889,
+    "eellipse": {
+        "a": 0.1,
+        "b": 0.1,
+        "cl": 95.0,
+        "theta": 0.0
+    },
+    "ra": 322.4154166666666,
+    "refs": [
+        "Bhandari2019"
+    ],
+    "z": 0.29116841268957017
+}
```

Comment:
Might be good to mention what a and b refer to under the "eelipse" and put a comment with the units (I assume it's the error on RA and Dec)

Reply:
I fear this can't be done in JSON files (only YAML).

But I have confirmed that it is documented in the `FRB` class.